server/test/services/openapi/v2/Swagger2SchemaGeneratorTest.java

```diff
@@ -0,0 +1,363 @@
+            .appendLine("          properties:")
+            .appendLine("            applicant_id:")
+            .appendLine("              type: integer")
+            .appendLine("              format: int32")
+            .appendLine("            application:")
+            .appendLine("              type: object")
+            .appendLine("              properties:")
+            .appendLine("                name:")
+            .appendLine("                  type: object")
+            .appendLine("                  properties:")
+            .appendLine("                    question_type:")
+            .appendLine("                      type: string")
+            .appendLine("                    first_name:")
+            .appendLine("                      type: string")
+            .appendLine("                      x-nullable: true")
+            .appendLine("                    last_name:")
+            .appendLine("                      type: string")
+            .appendLine("                      x-nullable: true")
+            .appendLine("                    middle_name:")
+            .appendLine("                      type: string")
+            .appendLine("                      x-nullable: true")
+            .appendLine("                sample_address_question:")
+            .appendLine("                  type: object")
+            .appendLine("                  properties:")
+            .appendLine("                    question_type:")
+            .appendLine("                      type: string")
+            .appendLine("                    city:")
+            .appendLine("                      type: string")
+            .appendLine("                      x-nullable: true")
+            .appendLine("                    corrected:")
+            .appendLine("                      type: string")
+            .appendLine("                      x-nullable: true")
+            .appendLine("                    latitude:")
+            .appendLine("                      type: number")
+            .appendLine("                      format: double")
+            .appendLine("                      x-nullable: true")
+            .appendLine("                    line2:")
+            .appendLine("                      type: string")
+            .appendLine("                      x-nullable: true")
+            .appendLine("                    longitude:")
+            .appendLine("                      type: number")
+            .appendLine("                      format: double")
+            .appendLine("                      x-nullable: true")
+            .appendLine("                    service_area:")
+            .appendLine("                      type: string")
+            .appendLine("                      x-nullable: true")
+            .appendLine("                    state:")
+            .appendLine("                      type: string")
+            .appendLine("                      x-nullable: true")
```

Comment:
**thought**: This is effectively an enum, not sure if that's worth calling out.

Reply:
I'm considering including enums, but not in swagger 2. openapi 3.x have better support and 3.1 specifically aligns with the json schema spec.